how-tos/disable-public-signups.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "Disable public signups"
+description:
+  "When you are in the early stages of your business, you might want to onboard
+  customers manually instead of having open signups."
+---
+
+PropelAuth already makes it easy for you to sign up users and their coworkers,
+but this is actually a case where you don't want that. The only users you want
+are ones you explicitly choose and no one else. Maybe these are only people on
+your waitlist, or maybe these are friends and family, or maybe it's people that
+you've given a demo to and are really excited about what you are building.
+
+To disable signups, go to the **Project Setting** section of your
```

Comment:
Recently moved to **Signup / Login** => **Settings**

Reply:
changed